src/custom/components/Header/index.tsx

```diff
@@ -265,19 +326,22 @@ export default function Header() {
           <NetworkCard />
           <HeaderElement>
             <UNIWrapper onClick={handleOnClickClaim}>
-              <VCowAmount active={!!account} style={{ pointerEvents: 'auto' }}>
+              <VCowAmount isClaimPage={isClaimPage} active={!!account} style={{ pointerEvents: 'auto' }}>
                 {claimTxn && !claimTxn?.receipt ? (
                   <Dots>
                     <Trans>Claiming vCOW...</Trans>
                   </Dots>
                 ) : (
                   <>
                     <CowProtocolLogo />
-                    <Trans>vCOW</Trans>
+                    <b>
+                      <Trans>Claim vCOW</Trans>
```

Comment:
Why this change? 
My idea was to say "Claim vCOW", but afaik Anna changed it to vCOW only so we always show the button.